src/index.html

```diff
@@ -15,8 +15,36 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="/" class="logo__img"><img src="./images/logo.png" alt="Moyo"></a>
+      <nav class="navigation">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="/apple" class="nav__link is-active">Apple</a>
```

Comment:
Don't use spaces in the `<a>` tag's `href` property. Anchor links should start with the `#` symbol if they are meant to be internal page links or be a valid URL if linking to external pages. For example, use `href="#apple"` for an internal link or `href="https://example.com/apple"` for an external link.